test/integrationtests/username_password_test.go

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+package integrationtests
+
+import (
+	"context"
+	"testing"
+
+	"github.com/AzureAD/microsoft-authentication-library-for-go/msal"
+)
+
+var labClientInstance = &labClient{}
+
+func AcquireTokenByUsernamePasswordCommonAAD(t *testing.T, user user, authority string, scope []string, appID string) {
```

Comment:
See my table driven comment below, but this should probably just be in a table driven test as the test code in the test loop instead of a separate function.